core/templates/pages/moderator-page/moderator-page.component.ts

```diff
@@ -16,6 +16,7 @@
  * @fileoverview Component for the Oppia moderator page.
  */
 
+import { ThreadMessage } from 'domain/feedback_message/ThreadMessage.model';
 require('base-components/base-content.directive.ts');
```

Comment:
Add a newline between import and require.

Reply:
Done